srcDoc/compilation.rst

```diff
@@ -500,14 +502,14 @@ examples:
        ./build.bat --platform x64Win64VS2012 --secure --dynamic
 
 -  Generation and compilation for a given architecture
-   (``x64Win64VS2012``) for Micro, specifying the RTIMEHOME.
+   (``x64Win64VS2012``) for *Connext DDS Micro*, specifying the RTIMEHOME.
 
    ::
 
        ./build.bat --platform x64Win64VS2012 --micro --rtimehome <PATH>
 
 -  Generation and compilation for a given architecture
-   (``x64Win64VS2012``) for Micro with security, debug,
+   (``x64Win64VS2012``) for *Connext DDS Micro* with security, debug,
```

Comment:
security, using debug mode and specifying

Reply:
sounds good

Reply:
done